ci/run_cudf_polars_polars_tests.sh

```diff
@@ -14,6 +14,24 @@ DESELECTED_TESTS=(
     "tests/docs/test_user_guide.py" # No dot binary in CI image
 )
 
+if [[ $(arch) == "aarch64" ]]; then
+    # The binary used for TPC-H generation is compiled for x86_64, not aarch64.
+    DESELECTED_TESTS+=("tests/benchmark/test_pdsh.py::test_pdsh")
+    # The connectorx package is not available on arm
+    DESELECTED_TESTS+=("tests/unit/io/database/test_read.py::test_read_database")
+    # The necessary timezone information cannot be found in our CI image.
```

Comment:
What timezone info are we missing?

Reply:
You can see the failures in https://github.com/rapidsai/cudf/actions/runs/11846396105/job/33015080248